cmake/ctest/build.ctest

```diff
@@ -171,6 +174,13 @@ set(_CONFIGURE_OPTIONS
     "-DUSE_CONAN=${USE_CONAN}"
     )
 
+if(CC)
+    list(APPEND _CONGIGURE_OPTIONS "-DCMAKE_C_COMPILER=${CC}")
+endif()
+if(CXX)
```

Comment:
Doesn't CMake respect the environment variables already?

Reply:
Yup, and ctest does too. I overcomplicated things because at the start of a travis build, there is an explicit export CC=gcc and export CXX=g++